Tests/Foundation/TestFileManager.swift

```diff
@@ -1364,7 +1364,7 @@ class TestFileManager : XCTestCase {
         }
         XCTAssertThrowsError(try fm.linkItem(atPath: "/tmp/t", toPath: "")) {
             let code = ($0 as? CocoaError)?.code
-            XCTAssertEqual(code, .fileNoSuchFile)
+            XCTAssertEqual(code, .fileReadNoSuchFile)
```

Comment:
These weren't immediately caused by this change, I think this was an effect of an underlying swift-foundation change